File: tool/writing/backend/color_text.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""color_text - 写作工具"""
from tool.writing.backend._base import handle_writing

TOOL_NAME = 'color_text'
SYS_PROMPT = "你是视觉文字排版专家。用颜色突出关键词、重点、角色、步骤或情绪，保持原文可读；HTML使用span color，Markdown使用可阅读的标记并说明颜色用途。"
TEMPERATURE = 0.4

def build_prompt(a, t):
    return "原文：\n"+t+"\n\n配色："+(a.get("color_scheme","主题色"))+"\n输出格式："+(a.get("format","html"))

def handle(body, ctx):
    handle_writing(body, ctx, TOOL_NAME, SYS_PROMPT, TEMPERATURE, build_prompt)

File: tools/writing/frontend/color_text.js
// ========== color_text.js - 工具定义 ==========
// 朱峰社区智能体无限 5.0.0 - 独立工具文件
// 自动生成自 4.1.8 tools.js（split_tools.js V3）
// 执行逻辑: tools/executor.js 中对应 case 分支

window.Tools = window.Tools || { allTools: {} };
if (!window.Tools.allTools['color_text']) {
    window.Tools.allTools['color_text'] = {
    color_text: {
        sys: "你是视觉文字排版专家。用颜色突出关键词、重点、角色、步骤或情绪，保持原文可读；HTML使用span color，Markdown使用可阅读的标记并说明颜色用途。",
        temp: 0.4,
        build: function(a, t) { return "原文：\n"+t+"\n\n配色："+(a.color_scheme||"主题色")+"\n输出格式："+(a.format||"html"); }
    },
    };
}
